api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -177,11 +177,9 @@ public static class AuthConfig {
 
   public static class WgsCohortExtractionConfig {
     public String serviceAccount;
-    public String serviceAccountTerraProxyGroup;
```

Comment:
Please do not remove flags that belong to workbench team. Please revert this change.

Reply:
oops didnt know that

Reply:
done